build-tools/packages/build-cli/test/commands/generate/buildVersion.test.ts

```diff
@@ -8,6 +8,9 @@ const test_tags = [
 	"client_v2.0.0-internal.1.0.0",
 	"client_v1.2.4",
 	"client_v1.2.3",
+	"client_v2.0.0-rc.1.0.0",
+	"client_v2.0.0-rc.1.0.1",
+	"client_v2.0.0-rc.2.0.0",
 	"build-tools_v0.5.2002",
```

Comment:
Is there an order to these? v2 internal is first, then v1, then v2 rc?
Perhaps order for readability if not ordered and comment either way.

Reply:
They're deliberately unordered; I'll add a comment.